This patch adds a FreeBSD-specific suffix to clang's version string.  This is
usually of the form "(yyyyddmm)", representing the date when the compiler was
last updated.

Introduced here: http://svnweb.freebsd.org/changeset/base/209107

Index: tools/clang/lib/Basic/Version.cpp
===================================================================
--- tools/clang/lib/Basic/Version.cpp
+++ tools/clang/lib/Basic/Version.cpp
@@ -124,8 +124,10 @@ std::string getClangFullVersion() {
   OS << "clang version " CLANG_VERSION_STRING " "
      << getClangFullRepositoryVersion();
 
+#ifdef CLANG_VENDOR_SUFFIX
+  OS << CLANG_VENDOR_SUFFIX;
+#elif defined(CLANG_VENDOR)
   // If vendor supplied, include the base LLVM version as well.
-#ifdef CLANG_VENDOR
   OS << " (based on LLVM " << PACKAGE_VERSION << ")";
 #endif
 
